python/main.py

```diff
@@ -22,10 +25,39 @@
 def root():
     return {"message": "Hello, world!"}
 
+@app.get("/items")
+def get_item():
+    try:
+        with open("items.json", "r") as f:
+            mydata = json.load(f)
+            return mydata
+    except FileNotFoundError:
+        return {}
+
 @app.post("/items")
-def add_item(name: str = Form(...)):
-    logger.info(f"Receive item: {name}")
-    return {"message": f"item received: {name}"}
+def add_item(name: str = Form(...), category: str = Form(...), image: UploadFile = File(...)):
+    logger.info(f"Receive item: {name}, Receive category: {category}, Receive image:{image.filename}")
+
+    # Hash the image using sha256, and save it with the name <hash>.jpg
+    file = image.file.read()
+    image_hash = hashlib.sha256(file).hexdigest()
+    filename = image_hash + ".jpg"
+    path = "images/" + filename
+    with open(path, "w") as f:
+        f.write(filename)
```

Comment:
The instruction maybe a little bit vague but could you save image binary as this file?